sys/phydat/phydat_str.c

```diff
@@ -64,6 +64,8 @@ const char *phydat_unit_to_str(uint8_t unit)
         case UNIT_TEMP_K:   return "K";
         case UNIT_LUX:      return "lx";
         case UNIT_M:        return "m";
+        case UNIT_M2:       return "sqm";
+        case UNIT_M3:       return "cbm";
```

Comment:
This doesn't work with `phydat_scale_to_str()`:

Proof by counterexample:

* 1000m² != 1km² == 1km * 1km == 1000m * 1000m == 1000000 m²
* 1000m³ != 1km³ == 1km * 1km * 1km == 1000m * 1000m * 1000m == 1000000000 m²